tool/tctl/common/resource_command.go

```diff
@@ -87,6 +92,16 @@ func (rc *ResourceCommand) Initialize(app *kingpin.Application, config *service.
 	rc.createCmd.Arg("filename", "resource definition file").Required().StringVar(&rc.filename)
 	rc.createCmd.Flag("force", "Overwrite the resource if already exists").Short('f').BoolVar(&rc.force)
 
+	rc.updateCmd = app.Command("update", "Update resource fields")
+	rc.updateCmd.Arg("resource type/resource name", `Resource to update
+	<resource type>  Type of a resource [for example: rc]
+	<resource name>  Resource name to update
+
+	Examples:
+	$ tctl update rc/remote`).SetValue(&rc.ref)
+	rc.updateCmd.Flag("set-labels", "Set labels").StringVar(&rc.labels)
```

Comment:
We don't sorry anything like this at the moment, using command line flags to update the resource. Is this a change we want to make? Add convenience flags?

Reply:
yes, why not? `--set-labels` and `--set-ttl` is common enough to update labels or TTL on resources